### ISSUE TITLE
File integrity failed after upgrade 16.0 to 16.1

### ISSUE THREAD

**Issue by guebli** [nextcloud/server#15627]
### Steps to reproduce
1.
2.
3.

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Server configuration

**Operating system**: freebsd 12.0

**Web server:** Apache 2.4

**Database:** Mariadb 10.2

**PHP version:** 7.1

**Nextcloud version:** 16.1 (see Nextcloud admin page)

**Updated from an older Nextcloud/ownCloud or fresh install:**

**Where did you install Nextcloud from:** freebsd ports

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
File from Accessibilty App can't be checking.

see below

Results
=======
- accessibility
	- FILE_MISSING
		- l10n/mk.js
		- l10n/mk.json

Raw output
==========
Array
(
    [accessibility] => Array
        (
            [FILE_MISSING] => Array
                (
                    [l10n/mk.js] => Array
                        (
                            [expected] => a2e44f768bc467705616e73de781c611c290a9aa58f8d48d1076cb2e4403894c68a6c613a7cd67ce39016213307171f02afd8a4206cd308c002a14ee1f8f9ad4
                            [current] => 
                        )

                    [l10n/mk.json] => Array
                        (
                            [expected] => d291395d459185fa98398792a5598f1b27c1cfb5d227b6999f88caefde4455acd2785a23b8e80a7e430be38eba7556545b24b59bff72f053189a560f9564f476
                            [current] => 
                        )

                )

        )

)

</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by dirigit** [comment]
To get rid of those unreliable and therefor very nagging integrity check messages I added 'integrity.check.disabled' => true permanent to config.

**Comment by kesselb** [comment]
It would be helpful if you share the error message.

**Comment by guebli** [comment]
Here is the message error:

Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- accessibility
	- FILE_MISSING
		- l10n/mk.js
		- l10n/mk.json

Raw output
==========
Array
(
    [accessibility] => Array
        (
            [FILE_MISSING] => Array
                (
                    [l10n/mk.js] => Array
                        (
                            [expected] => a2e44f768bc467705616e73de781c611c290a9aa58f8d48d1076cb2e4403894c68a6c613a7cd67ce39016213307171f02afd8a4206cd308c002a14ee1f8f9ad4
                            [current] => 
                        )

                    [l10n/mk.json] => Array
                        (
                            [expected] => d291395d459185fa98398792a5598f1b27c1cfb5d227b6999f88caefde4455acd2785a23b8e80a7e430be38eba7556545b24b59bff72f053189a560f9564f476
                            [current] => 
                        )

                )

        )

)

**Comment by skjnldsv** [comment]
Fixed since a while #15623
